<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComment</class>
 <widget class="QDialog" name="AddComment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>473</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 35, 35);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="Top">
     <item>
      <widget class="QTreeWidget" name="Comment_Display">
       <property name="minimumSize">
        <size>
         <width>384</width>
         <height>373</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 40, 40);</string>
       </property>
       <column>
        <property name="text">
         <string>Timestamp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comment</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="RightSide">
       <item>
        <layout class="QHBoxLayout" name="TopRow">
         <item>
          <widget class="QLabel" name="Event_Label">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>16</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Event#:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Event_Display">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>91</width>
             <height>16</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>E000000000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Timestamp_Label">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>91</width>
             <height>16</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Timestamp:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Timestamp_Display">
           <property name="minimumSize">
            <size>
             <width>161</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>161</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>00/00/0000 00:00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="CallType_Label">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Call Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CallType_Input">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="RP_Display">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>RP:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="RP_Input">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Phone_Label">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Phone:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Phone_Input">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Location_Label">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Location: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Location_Input">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Buttons">
         <item>
          <widget class="QPushButton" name="Close_Button">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Update_Button">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="Comment_Input">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="AddComment_Button">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Add Comment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CompleteCall_Button">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Complete Call</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
